docs/en/observability/tab-widgets/filebeat-logs/content.asciidoc

```diff
@@ -14,18 +14,21 @@ filebeat.inputs:
   paths: /path/to/logs.json
   parsers:
     - ndjson:
-      keys_under_root: true
-      overwrite_keys: true
-      add_error_key: true
-      expand_keys: true
+      overwrite_keys: true <2>
+      add_error_key: true <3>
+      expand_keys: true <4>
```

Comment:
Removed `keys_under_root` as that is now the default. This can be changed with `target`, but I didn't think it was necessary to list it here as we're not suggesting changing the default.

Reply:
@bmorelli25 do you know which version introduced this change and does it applies to other keys that are injected into the document ?
In the ready-to-use examples that I currently have (initially built for 8.5.3, but now upgraded to 8.6.0):
- `fields_under_root`: https://github.com/elastic/apm-contrib/blob/b99ee6225f01bad97901396d63a2f29d31a57f2a/apm-agent-java/log-ingest/01-filebeat.yml#L20 (for plain-text logs)
- `keys_under_root` https://github.com/elastic/apm-contrib/blob/b99ee6225f01bad97901396d63a2f29d31a57f2a/apm-agent-java/log-ingest/02-filebeat.yml#L13 (for nd-json logs)

Reply:
For nd_json logs, `keys_under_root` doesn't do anything and never has; a documentation error made it appear necessary. In 8.2, the documentation was updated to show `target` instead of `keys_under_root`. If `target` is not set, fields go to the root of the event by default ([source](https://github.com/elastic/beats/issues/29422#issuecomment-1105361562)).

For plain text logs, you are correct. `fields_under_root` is needed to store custom fields as top-level fields in the output document instead of being grouped under a `fields` sub-dictionary ([source](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html#fields-under-root-filestream)).